nav2_msgs/srv/LoadMap.srv

```diff
@@ -0,0 +1,22 @@
+# Type definitions
+uint8 TYPE_FILE=0
+uint8 TYPE_URL=1
+
+# Type of map resource
+uint8 type
+
+# ID of map resource
+string map_id
+
+---
+
+# Result code defintions
+uint8 RESULT_SUCCESS=0
+uint8 RESULT_MAP_DOES_NOT_EXIST=1
```

Comment:
If they're not implemented yet as a `TODO` is there harm in removing the other codes and adding them back in when its going to be implemented?